src/GenerateLibs.fsx

```diff
@@ -16,6 +16,7 @@ let girFiles =
         "GdkPixbuf-2.0.gir"
         "Gio-2.0.gir"
         "GLib-2.0.gir"
+        "GModule-2.0.gir"
```

Comment:
This was the only library that was previously getting code generated but wasn't listed explicitly here. I'm not sure whether this is really needed though as it doesn't look like this library is used as a project dependency by any of the other libraries or published as a NuGet package.

Reply:
Please remove this line as there is no GModule-2.0 project / nuget. I don't know if / when we are going to need this project.